protostar/profiler/contract_profiler.py

```diff
@@ -0,0 +1,319 @@
+# pylint: disable=invalid-name
```

Comment:
nitpick: what name is invalid? could we avoid global linter disable comment here?

Reply:
f*ck this lint, can we disable it globally?